ontology/yaml/resources/HVAC/entity_types/DOAS.yaml

```diff
@@ -16,6 +16,23 @@
 ### Canonical Types  ###
 ########################
 
+DOAS_HTWHLSTC_SFVSC_EFVSC_EDM_SDM_RFC_SFC_FDPM2X_RTM_FPHC:
+  guid: "f669b241-06bd-40ba-bc14-98a08989695a"
+  description: "DOAS with supply and return fans, heatwheel and heating and cooling valves."
+  is_canonical: true
+  implements:
+  - DOAS
+  - HTWHLSTC
+  - SFVSC
+  - EFVSC
+  - EDM
+  - SDM
+  - RFC
+  - SFC
+  - FDPM2X
+  - RTM
+  - FPHC
```

Comment:
FPHC is typically used for radiant heating/cooling (radiant beams, panels, etc.) -- why not use CHWSC and HWSC ?

Reply:
Thank you for input  - have used both CHWSC and HWSC to cover heating and cooling valve percentage commands